src/templates/ApiReference.js

```diff
@@ -232,7 +232,15 @@ const headerOptions = {
   treatIdAsHref: true,
   LinkComponent: DocsLink,
 };
-const ApiRefH1 = makeLinkedHeader(H1, headerOptions);
+/* Hack to make it look appear as if we had a column-gap
+4rem between colums and its h1 on a large screen */
+const XlH1 = styled(H1)`
+  @media (${MEDIA_QUERIES.gtXlDesktop}) {
+    margin-left: 4rem;
```

Comment:
I notice the gap gets misaligned at some screen sizes:
<img width="318" alt="Screen Shot 2020-04-09 at 4 47 37 PM" src="https://user-images.githubusercontent.com/1551487/78939318-e8017180-7a81-11ea-9646-7d9a5d9ce4dc.png">
Maybe instead of a fixed margin, the H1 in `ApiReference.js` should move within the grid? Might mean adding a new row though.

Reply:
Good call!

Reply:
Added a new row to surround `<H1/>` and removed `<Container/>` from ApiReference since ApiReference shouldn't have `max-width` which was coming form `<Container/>` previously.